centraldogma/base_client.py

```diff
@@ -23,15 +23,37 @@
 class BaseClient:
     PATH_PREFIX = "/api/v1"
 
-    def __init__(self, base_url: str, token: str, http2: bool = True, **configs):
+    def __init__(
+        self,
+        base_url: str,
+        token: str,
+        http2: bool = True,
+        retries: int = 0,
+        max_connections: int = 100,
+        max_keepalive_connections: int = 20,
```

Comment:
The default values seem too large in HTTP/2.
```suggestion
        max_connections: int = 10,
        max_keepalive_connections: int = 2,
```

Reply:
As described in PR, default values are same with httpx https://www.python-httpx.org/advanced/resource-limits/
Since these values are proper in general purpose, I thought it is not that too large. But, from point of view of CentralDogma, your suggestion makes sense. Changed!